docs/credits.md

```diff
@@ -3,121 +3,6 @@
 ## Sponsors
 <!-- readme: sponsors -start -->
 <table>
-<tr>
```

Comment:
Did you mean to delete this?

Reply:
I just edited zh-CN.json, other files may be modified by bot, I forgot to check when submitting. I don't know how to modify this pull request.